src/components/DownloadButton.vue

```diff
@@ -91,7 +91,7 @@ export default {
   },
   data() {
     const savedFormatExtension =
-      local.get(LS_DOWNLOAD_FORMAT_EXTENSION_KEY) ?? null
+      local.getItem(LS_DOWNLOAD_FORMAT_EXTENSION_KEY) ?? null
```

Comment:
Would using `useStorage` composable be an overkill here?

Reply:
I guess not, I'd just like to avoid doing that refactor in this PR as well if that's okay? We can open a separate issue for it :slightly_smiling_face:

Reply:
Sure, I just remembered looking at this PR previously and trying to understand the difference between the two. Then I saw that `useStorage` is actually using `local` internally